charts/rd-professional-api/values.preview.template.yaml

```diff
@@ -16,7 +16,4 @@ java:
   keyVaults:
     rd:
       secrets:
-        - s2s-secret
-
-
-
```

Comment:
have we changed anything here?

Reply:
for some reason this is showing as a deletion but it is still there if you view the raw file

Reply:
if no change please revert this file.